### PR TITLE
Contacts Updated

### PR DESCRIPTION
**Pull request by Surya1231**
Contacts And UI Updated

### CODE REVIEW

**Comment by rishabhKalakoti** [comment]
You need to resolve conflicts first

File: index.html [hunk 1]
```diff
@@ -53,6 +52,17 @@
...
+
+
+            <ul class="list-unstyled components" style="padding-top:0">
+            <li>
+                <a href="https://www.facebook.com/codeshows/" target="_blank"><i class="fab fa-facebook-f" style="margin-right:5px; margin-left:5px"></i> Facebook</a>
+            </li>
+
+            <li>
+              <a href="https://t.me/codeshows" target="_blank"><i class="fab fa-telegram-plane"></i> Telegram</a>
+            </li>
```
Comment (rishabhKalakoti) [inline index.html:61]: Just keep the icons for facebook and telegram in a line. Not the text

**Review by rishabhKalakoti — CHANGES_REQUESTED** [review]

**Comment by Surya1231** [comment]
How to resolve this conflicts??

**Comment by Surya1231** [comment]
The resolve conflicts button in disabled? what is conflict ? and how to resolve it??

**Comment by rishabhKalakoti** [comment]
Conflicts arise when you don't have the updated copy of the repository and your updates clash with another that was made in the meantime.
They can be resolved in two ways:
1. Github GUI:
Make a PR from main repo (codeshows/codeshows.gitub.io) to your fork (<your_user_name>/codeshows.github.io) and merge it (using the GitHub GUI).
The conflicting files will be opened for editing containing both the copies of the updated sections. They have to be done manually. (some tools can also be used but have to be installed).
2.  Command Line:
run,
`git add remote upstream https://github.com/codeshows/codeshows.github.io`
`git fetch upstream`
`git rebase upstream/master`
The conflicting files have to be opened manually and the final code chosen.

References:
https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository

**Comment by rishabhKalakoti** [comment]
Conflicts arise when you don't have the updated copy of the repository and your updates clash with another that was made in the meantime.
They can be resolved in two ways:
1. Github GUI:
Make a PR from main repo (codeshows/codeshows.gitub.io) to your fork (<your_user_name>/codeshows.github.io) and merge it (using the GitHub GUI).
The conflicting files will be opened for editing containing both the copies of the updated sections. They have to be done manually. (some tools can also be used but have to be installed).
2.  Command Line:
run,
`git add remote upstream https://github.com/codeshows/codeshows.github.io`
`git fetch upstream`
`git rebase upstream/master`
The conflicting files have to be opened manually and the final code chosen.

References:
https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository

**Comment by Surya1231** [comment]
conflicts resolved now please merge

**Pull request closed without merging**